Directory.Build.props

```
@@ -37,7 +37,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="coverlet.msbuild" Version="6.0.0">
+    <PackageReference Include="coverlet.msbuild" Version="6.0.1">
```

```suggestion
    <PackageReference Include="coverlet.msbuild" Version="6.0.2">
```

@ArieGato This can be merged after bump.